### DISCUSSION TITLE
Clickable headers

### DISCUSSION THREAD

**Discussion by Dee66** [nick-keller/react-datasheet-grid#255] [Q&A]
I've read through the Q&A but not found a suitable answer.

I'm trying to make the header row (well the columns of the header row) clickable to implement sorting, having trouble capturing the click event though.
Also searching / filtering by column, is that possible?

**Comment by nick-keller** [comment]
Hey @Dee66,
You can implement searching and filtering yourself with no issue. You simply need to do the filtering / searching first, and then pass the result to `value={result}` to only show the filtered rows.

If you want to implement this in the headers that is not possible for the moment because the click event is high-jacked, but I'll try to release a fix.

**Comment by Dee66** [comment]
Hey Nick, thanks for the quick response.

I'm already using the value attribute to set the data, problem is the
criteria needs to be set (as the column header that was clicked).
Can't see any other way than somehow capturing the cell click, recon you'd
be able to override (well catch) the event?

**Comment by Dee66** [comment]
Hey Nick, any news on them sortable (well clickable) headers?
